Cargo.toml

```diff
@@ -8,6 +8,7 @@ version = "0.10.1"
 
 [workspace.dependencies]
 sylvia-derive = { version = "0.10.1", path = "sylvia-derive" }
+cw-orch = { git = "https://github.com/abstractsdk/cw-orchestrator", branch = "update/relax-bound-on-derive" }
```

Comment:
Obviously, this would need to be merged and tagged before we can accept a PR, but that should be doable.

Reply:
The bigger question is why we have to tie Sylvia (even as an optional dependency) to one version of cw-orch. (eg 0.10x sylvia uses cw-orch 0.22, 0.11.x sylvia uses cw-orch 0.23). Maybe someone with more macro magic than I can see how to solve that.